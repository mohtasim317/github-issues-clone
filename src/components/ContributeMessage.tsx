import React, { FC } from "react";

const ContributeMessage: FC = () => {
  return (
    <div className="contribute-message">
      <h4 className="contribute">
        <span role="img" aria-label="hand waving emoji">
          ðŸ‘‹
        </span>
        Want to contribute to facebook/react?
      </h4>
      <p className="contribute">
        If you have a bug or an idea, read the contributing guidelines before
        opening an issue.
      </p>
    </div>
  );
};

export default ContributeMessage;
