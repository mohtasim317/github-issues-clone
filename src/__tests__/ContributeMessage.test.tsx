import React from "react";
import renderer from "react-test-renderer";
import ContributeMessage from "../components/ContributeMessage";

describe("ContributeMessage Component Snapshot", () => {
  it("renders a Contribute Message with instructions on how to submit to the React repo", () => {
    const component = renderer.create(<ContributeMessage />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
