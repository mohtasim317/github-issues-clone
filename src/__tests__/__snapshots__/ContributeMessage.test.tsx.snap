// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContributeMessage Component Snapshot renders a Contribute Message with instructions on how to submit to the React repo 1`] = `
<div
  className="contribute-message"
>
  <div
    className="contribute-first-line"
  >
    <h4
      className="contribute contribute-one"
    >
      <span
        aria-label="hand waving emoji"
        className="hand-emoji"
        role="img"
      >
        ðŸ‘‹
      </span>
      <div
        className="contribute-first-line-text"
      >
        Want to contribute to facebook/react?
      </div>
    </h4>
  </div>
  <p
    className="contribute contribute-two"
  >
    If you have a bug or an idea, read the
    <span>
      <a
        className="contributing-link"
        href="/"
      >
         
        contributing guidelines
         
      </a>
    </span>
    before opening an issue.
  </p>
</div>
`;
